models/Series.js

```diff
@@ -1,40 +1,40 @@
-"use strict";
-
-var mongoose = require('mongoose');
-var mongodb = require('common/makeCredential')('mongodb');
-var db = mongoose.createConnection(mongodb);
+import mongoose from 'mongoose';
+import makeCredential from 'common/makeCredential';
+const mongodb = makeCredential('mongodb');
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).